README.md

```diff
@@ -66,6 +66,13 @@ The server will run at http://localhost:3000/.
 
 You can now deploy the contents of the `build` directory as any other node.js app.
 
+## Running inside a docker container
+
+1. Make sure your `.env` file is populated with correct values.
+2. Run ```docker build -t matrix-blog-admin .```
+
+You can now start the container with ```docker run -p 3000:3000 matrix-blog-admin``` and access it at http://172.17.0.2:3000/ (or similar).
```

Comment:
Why `172.17.0.2` and not localhost? If you want to highlight that the server is accessible from the network, maybe add a sentence about using "your machine's IP address" or something.